datadog/resource_datadog_monitor.go

```diff
@@ -366,6 +378,40 @@ func resourceDatadogMonitor() *schema.Resource {
 									},
 								},
 							},
+							"custom_schedule": {
+								Description: "Configuration options for the custom schedules. If `start`is omitted, the monitor creation time will be used.",
+								Type:        schema.TypeList,
+								Optional:    true,
+								Elem: &schema.Resource{
+									Schema: map[string]*schema.Schema{
+										"recurrences": {
```

Comment:
Nit - I think this would read better as just `recurrence`, since each entry is a single one. If we had multiple it'd be confusing for them to be plural, e.g.:
```
resource "datadog_monitor" "foo" {
  name = "foo"
  type = "metric alert"
  message = "a message"
  priority = 3
  query = "avg(last_1h):avg:system.load.5{*} > 0.5"
  monitor_thresholds {
	critical = "0.5"
  }
  scheduling_options {
	custom_schedule {
		recurrences {
			rrule = "FREQ=DAILY;INTERVAL=1;WEEKDAY=MO,TU"
			timezone = "America/New_York"
			start = "2023-11-10T12:31:00"
		}
               recurrences {
			rrule = "FREQ=DAILY;INTERVAL=1;WEEKDAY=SA,SU"
			timezone = "America/New_York"
			start = "2023-11-10T12:31:00"
		}
	}
  }
}
```

Reply:
is it possible to have multiple recurrences? the schema has `MaxItems:    1,` defined on the recurrences schema.

Reply:
> is it possible to have multiple recurrences? the schema has `MaxItems: 1,` defined on the recurrences schema.

Not at the moment, but we could potentially support more than 1 later.